src/Sitemap/SitemapUrl.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace OHMedia\PageBundle\Sitemap;
+
+class SitemapUrl
+{
+    public readonly float $priority;
+
+    public function __construct(
+        public readonly string $path,
+        public readonly \DateTimeInterface $lastmod
```

Comment:
`lastmod` is also generally ignored because developers abuse it by putting fake info there. With the page bundle (and dynamic content like in the event bundle) we'll be using our `updated_at` times so these will be accurate.